docs/build-source.md

```diff
@@ -7,14 +7,14 @@ Building Azure SignalR Service SDK from source allows you tweak and customize th
 
 Building Azure SignalR Service SDK requires:
 
-* Visual Studio **2019 Preview**. <https://visualstudio.com>
+* Visual Studio **2019 (Preview)**. <https://visualstudio.com>
```

Comment:
2019 is already released

Reply:
Right. But Preview can experience latest features. Since we're now keep up with latest .NET Core SDK and suggest to work with latest Visual Studio.